preferences/legacy/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ../components/interfacemodel-e1000e
```

Comment:
As discussed lets use `e1000` for more coverage.

Reply:
Done.